invenio/modules/oauthclient/contrib/orcid.py

```diff
@@ -127,17 +127,20 @@ def account_info(remote, resp):
 @session_manager
 def account_setup(remote, token):
     """Perform additional setup after user have been logged in."""
-    from invenio.modules.accounts.models import User, UserEXT
-    from invenio.ext.sqlalchemy import db
-    from ..handlers import token_session_key
+    import json
+    import requests
 
     from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
 
-    import json
-    import requests
+    from invenio.modules.accounts.models import User, UserEXT
+    from invenio.ext.sqlalchemy import db
+    from ..handlers import token_session_key
 
     orcid = session.get(token_session_key(remote.name) +
-                        "_account_info").get("external_id")
+                        "_account_info", {}).get("external_id")
+
+    if not orcid:
```

Comment:
When this happens is it considered an error? Shall we add a logging?

Alternative:
```python
try:
     orcid = session.get(token_session_key(remote.name) +
                        "_account_info").get("external_id")
except KeyError:
    current_app.logger.exception("No ORCID available in user session.")
    return
```
WDYT @jmartinm @lnielsen?